docs/src/modules/components/MarkdownElement.js

```diff
@@ -67,14 +67,14 @@ const Root = styled('div')(({ theme }) => ({
     fontFamily: theme.typography.fontFamilySystem,
     fontWeight: 700,
     color: theme.palette.mode === 'dark' ? theme.palette.grey[100] : theme.palette.grey[900],
-    margin: '40px 0 8px',
+    margin: '40px 0 4px',
   },
   '& h3': {
     ...theme.typography.h6,
     fontFamily: theme.typography.fontFamilySystem,
-    fontWeight: 700,
+    fontWeight: 600,
```

Comment:
What's the impact on performance?

Reply:
I can't say, does it concern you?

Reply:
I think that we should know what we are giving up in exchange. AFAIk, Kerformance is the why behind "trying to constrain ourselves with 3 weights"

Reply:
Ok, I don't know about performance, let's see Jun's thoughts.

Reply:
@danilo-leal we currently load 400, 500, 700 so 600 does not work.

Reply:
But Olivier's comment is worth thinking about. Otherwise, there will be a lot of font-weights, and hard to keep the consistency. Anyway, I'll let you decide on this @danilo-leal.

Reply:
I'll take the risk and have the 600 weight. It was missing to me especially for h3 and h4 headings where h2 is already on bold and 500 is just too thin when the font size is bigger than 20px.